src/main/kotlin/net/atos/zac/app/klant/model/bedrijven/RestKlantenAdres.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.app.klant.model.bedrijven
+
+class RestKlantenAdres(
```

Comment:
Can this be `data class`?

Reply:
probably yes. will have a look. just those inherited classes cannot be data classes. or at least, better not because data classes and inheritance don't work well together

Reply:
fixed this